webapp/channels/src/components/admin_console/team_channel_settings/group/group_list.tsx

```diff
@@ -66,9 +67,21 @@ const GroupList = ({
         <AbstractList
             header={<Header/>}
             renderRow={renderRow}
+            emptyListText={isModeSync ? messages.emptyListModeSync : messages.emptyList}
```

Comment:
This used to be done in the index file for `AbstractList` which seemed really odd to me and was confusing TypeScript's ability to check the type of props, so I moved it here to make that easier